trino-lb/src/scaling/mod.rs

```diff
@@ -540,7 +550,11 @@ impl Scaler {
         Ok((cluster_name, current_state))
     }
 
-    #[instrument(name = "Scaler::apply_target_states", skip(self))]
+    #[instrument(
+        name = "Scaler::apply_cluster_target_state",
+        skip(self, cluster),
+        fields(cluster_name = cluster.name)
```

Comment:
note: OpenTelemetry recommends naming fields using dots, in this case `cluster.name`. I think the `#[instrument]` attribute can use a shortcut like this:

```suggestion
        fields(%cluster.name)
```